tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/validation/ValidationBug427432Test.xtend

```diff
@@ -31,7 +31,7 @@ class ValidationBug427432Test extends AbstractXtendTestCase {
 		'''
 			class Foo {
 			
-				val x = 1
+				static val x = 1
```

Comment:
Without the static keyword the switch case constant values aren't evaluated anymore, so the validation does not work anymore for this case. This is ok since the compiler now does not create a switch, but an if-else chain. I removed the static modifier and changed the assertion to "no errors".